modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,258 @@
+
+The QNAP ready-to-use ownCloud installation should only be changed within very tight limits. If you change the database or web server configuration, the installation will likely break.
+
+All configuration should happen in the Web interface. However, command-line configuration via `occ` commands is availabe.
+
+== Minimum Hardware Requirements
+
+For ownCloud to properly run on a QNAP NAS device, at least 2 GB of RAM are necessary. However, we recommend at least 4 GB for a smooth operation.
+
+== Preparation
+
+If you haven't done so already, perform the following steps on you QNAP system:
+
+. Create a storage pool in addition to the system volume. You can combine hard disks and solid state disks in RAID groups for flexible storage. This pool can be resized later if necessary. Go to menu:Main Menu[Storage & Snapshots] or click on the icon.
++
+image:qnap/storage_pool.png[Storage Pool Creation]
+
+. Via the App Center install Container Station version 2.0 or higher.
++
+image:qnap/qnap-app_center.png[QNAP App Center with Container Station installed]
+
+== Installation of ownCloud
+
+In the App Center search for ownCloud and click btn:[Install]. Confirm that you want to install this app. During installation, you'll see a progress bar.
+
+After a successful installation, ownCloud will show up on your desktop and in the App Center under menu:QNAP Store[My Apps]. Click btn:[Open] under the ownCloud icon and proceed with the configuration.
+
+image:qnap/qnap_main_menu.png[QNAP Main Menu and ownCloud installed]
+
+== Configuration
+
+=== First Steps
+
+Log in as user `admin`. The default password is `admin` until you change it. In the upper right-hand corner, click on menu:admin[Settings]. In the left hand navigation bar, you find personal configuration options and administration-specific options. Under menu:Personal[General] you can change the password of your admin user.
+
+For more information on the web interface, refer to the xref:user_manual:webinterface.adoc[Web UI] documentation.
+
+=== Configure your Email.
+
+Email is an essential tool in ownCloud for notifying users, user registration, password resets, etc. Logged-in as admin, you can configure email in the menu:Admin[General] settings.
+
+For more information, refer to section xref:configuration/server/email_configuration.adoc[Email Configuration].
+
+=== Cron
+
+ownCloud performs regular tasks in the background. In order to schedule them, a mechanism called "cron" is used. It's already set up for you, so you don't need to do anything. Make sure not to change cron-related settings unless you have a very good reason to do so.
+
+NOTE: In general, we recommend to keep the default settings in the admin section.
+
+== Users
+
+A key feature of ownCloud is to allow users to share files. In the free community edition you can create up to five users, including an admin user, and grant access to an unlimited number of guest users.
+
+=== Prerequisites:
```

Comment:
IMO this is prerequisites for Guest users. If you don't need/want Guest users, then you don't need to do this.

Reply:
Right, but it's a feature. Some rephrasing...